src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,9 @@
 public class Application {
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery[] balls = { new Lottery(), new Lottery(), new Lottery() };
```

Comment:
Instead of creating multiple `Lottery` objects, you should create a single `Lottery` object and use it to generate multiple `Ball` objects using the `getRandomBall()` method. This will align with the task requirement to create multiple `Ball` objects.